fxmanifest.lua

```diff
@@ -25,7 +25,7 @@ files {
     'html/js/script.js',
     'html/css/style.css',
     'html/img/radio.png',
-    'config/shared.lua',
+    'config/*.lua',
```

Comment:
```suggestion
    'config/shared.lua',
    'config/client.lua',
```
When invariably someone throws some server specific stuff in here we don't need to include that to the client.